src/modules/adempiere-api/api/extensions/adempiere/import-file-loader/index.js

```diff
@@ -0,0 +1,208 @@
+/************************************************************************************
+ * Copyright (C) 2012-2023 E.R.P. Consultores y Asociados, C.A.                     *
+ * Contributor(s): Elsio Sanchez elsiosanchez15@outlook.com                         *
+ * This program is free software: you can redistribute it and/or modify             *
+ * it under the terms of the GNU General Public License as published by             *
+ * the Free Software Foundation, either version 2 of the License, or                *
+ * (at your option) any later version.                                              *
+ * This program is distributed in the hope that it will be useful,                  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of                   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                     *
+ * GNU General Public License for more details.                                     *
+ * You should have received a copy of the GNU General Public License                *
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.            *
+ ************************************************************************************/
+
+import { Router } from 'express';
+import { getLookupItemFromGRPC } from '@adempiere/grpc-api/src/utils/userInterfaceFromGRPC';
+import { convertEntitiesListFromGRPC } from '../util/convertData';
+
+function getImportFormatFromGRPC (importFormatToConvert) {
+  if (!importFormatToConvert) {
+    return undefined;
+  }
+  return {
+    id: importFormatToConvert.getId(),
+    uuid: importFormatToConvert.getUuid(),
+    name: importFormatToConvert.getName(),
+    description: importFormatToConvert.getDescription(),
+    tableName: importFormatToConvert.getTableName(),
+    formatType: importFormatToConvert.getFormatType(),
+    separatorCharacter: importFormatToConvert.getSeparatorCharacter(),
+    formatFields: importFormatToConvert.getFormatFields()
+  };
+}
+
+function resourceReferenceFromGRPC (resourceReferenceToConvert) {
+  if (!resourceReferenceToConvert) {
+    return undefined;
+  }
+  return {
+    resourceId: resourceReferenceToConvert.getResourceId(),
+    resourceUuid: resourceReferenceToConvert.getResourceUuid(),
+    fileName: resourceReferenceToConvert.getFileName(),
+    fileSize: resourceReferenceToConvert.getFileSize(),
+    description: resourceReferenceToConvert.getDescription(),
+    textMsg: resourceReferenceToConvert.getTextMsg(),
+    contentType: resourceReferenceToConvert.getContentType()
+  };
+}
+
+module.exports = ({ config }) => {
```

Comment:
please convert to typescript and change path from `src/modules/adempiere-api/api/extensions/adempiere/import-file-loader/index.js` to `src/modules/adempiere-api/api/extensions/adempiere/import-file-loader.ts` (without folder)